src/Glpi/Form/QuestionType/AbstractQuestionType.php

```diff
@@ -88,6 +89,28 @@ public function renderAdministrationOptionsTemplate(?Question $question): string
         return ''; // No options by default
     }
 
+    #[Override]
+    public function renderAnswerTemplate($answer): string
```

Comment:
All overrides should also have typed parameters. There are multiple classes to update.
```suggestion
    public function renderAnswerTemplate(mixed $answer): string
```